packages/server/src/api/routes/tests/appSync.spec.ts

```diff
@@ -0,0 +1,31 @@
+import * as setup from "./utilities"
+import { roles, db as dbCore } from "@budibase/backend-core"
+
+describe("/api/applications/:appId/sync", () => {
+  let request = setup.getRequest()
+  let config = setup.getConfig()
+  let app
+
+  afterAll(setup.afterAll)
+
+  beforeAll(async () => {
+    app = await config.init()
+    // create some users which we will use throughout the tests
+    await config.createUser({
+      email: "sync1@test.com",
+      roles: {
+        [app._id!]: roles.BUILTIN_ROLE_IDS.BASIC,
+      },
+    })
+  })
+
+  async function getUserMetadata() {
+    const { rows } = await config.searchRows(dbCore.InternalTable.USER_METADATA)
+    return rows
+  }
+
+  it("make sure its empty initially", async () => {
```

Comment:
Does the test name need updated? Looks like it used to model a test where the results were `[]`

Reply:
I would be a fan of that approach, to have an empty state + a test after sync

Reply:
There is no empty state anymore - the app starts in a sync'd state.